CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Mattermost Desktop Application Changelog
 
+## Release v3.4.1
+
+Release date: TBD
+
+### Bug Fixes
+
+#### macOS
```

Comment:
should we change "OS X" to "macOS" elsewhere in the changelog, for consistency?

Also, how would you feel using "Mac" instead?